ARExperiment/ARExperiment.xcodeproj/xcuserdata/berta.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please create a gitignore so that you don't commit this stuff

Reply:
done